config/webpack.common.js

```diff
@@ -1,8 +1,8 @@
 const webpack = require('webpack');
 const path = require('path');
+const AssetsWebpackPlugin = require('assets-webpack-plugin');
```

Comment:
From looking at the documentation, `yarn.lock` is a auto-generated file and shouldn't be edited.